packages/php/blueprint/tests/TestCase.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Automattic\WooCommerce\Blueprint\Tests;
+
+use PHPUnit\Framework\TestCase as PHPUnitTestCase;
+
+require_once __DIR__.'/helpers.php';
+
+abstract class TestCase extends PHPUnitTestCase
+{
+	protected bool $load_wp = false;
+	public function __construct() {
+		if ( $this->load_wp ) {
+		}
```

Comment:
This is an empty `if` block. Do we need to do something when wp is loaded?

Reply:
Thank you for the catch! Updated in [Remove unused construct code](https://github.com/woocommerce/woocommerce/pull/49763/commits/0dead27effa18ee0f227557b05bc696abe4812c2)